deep_daze/deep_daze.py

```diff
@@ -110,7 +124,10 @@ def __init__(
             image_width=512,
             loss_coef=100,
             theta_initial=None,
-            theta_hidden=None
+            theta_hidden=None,
+            lower_bound_cutout=0.1, # should be smaller than 0.8
```

Comment:
Would these paramaters be useful for the CLI to use?

Reply:
Yes, the lower_bound should be useful.

Reply:
I can work on this one as well if you havent yet.

Reply:
https://github.com/NotNANtoN/deep-daze/pull/1/files